src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -347,18 +341,24 @@ public final void setSortImportsInGroupAlphabetically(boolean value) {
     }
 
     /**
-     * Setter to specify format of order declaration customizing by user.
+     * Setter to specify list of import groups.
      *
-     * @param inputCustomImportOrder
+     * @param rules
      *        user value.
      * @since 5.8
      */
-    public final void setCustomImportOrderRules(final String inputCustomImportOrder) {
-        if (!DEFAULT_CUSTOM_IMPORT_ORDER_RULES.equals(inputCustomImportOrder)) {
-            for (String currentState : GROUP_SEPARATOR_PATTERN.split(inputCustomImportOrder)) {
```

Comment:
We didn't need this condition any longer, this accounted for a blank string which is not an issue now.